### ISSUE TITLE
Converting loaded assets to use toon shaded materials needs an example

### ISSUE THREAD

**Issue by alice-i-cecile** [tbillington/bevy_toon_shader#3]
This is surprisingly tricky, but essential for using this crate in more serious games.

**Comment by alice-i-cecile** [comment]
The best approach I've been able to find is https://github.com/DGriffin91/bevy_mod_standard_material. This would make a great example.

**Comment by tbillington** [comment]
Yep can make an example based on bevy_mod_standard_material.

I assume at some point using a different material for spawned scenes would be a "bevy" level feature.

**Comment by alice-i-cecile** [comment]
Yep, this should be feasible with the asset preprocessing work that Cart's been working on. Figuring out the pattern / an example here will give us something to ~~steal~~ lovingly borrow as an example for upstream too.

**Comment by mattoni** [comment]
Any examples on how to use this on a loaded gltf scene?
